lib/ckb/address.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module CKB
+  class Address
+    attr_reader :pubkey
+
+    MODE_TESTNET = "testnet"
+    MODE_MAINNET = "mainnet"
+    MODE_CUSTOM = "custom"
```

Comment:
Isn't this used anywhere? If so can we remove it for now?

Reply:
Move to `ckb/mode.rb`

Reply:
Good move. But still, is `custom` mode even used? And what does that mean? Connecting to a custom node (URL)? If so, that node is still either testnet (including private network) or mainnet.

Reply:
Yes, it's useless now, removed.